pkg/cluster/nodes/node.go

```diff
@@ -141,17 +144,17 @@ func (n *Node) LoadImages() {
 	// TODO(bentheelder): this is a bit gross, move this logic out of bash
 	if err := n.Command(
 		"/bin/bash", "-c",
-		`docker images --format='{{.Repository}}:{{.Tag}}' | grep -v amd64 | xargs -L 1 -I '{}' /bin/bash -c 'docker tag "{}" "$(echo "{}" | sed s/:/-amd64:/)"'`,
+		fmt.Sprintf(`%s images --format='{{.Repository}}:{{.Tag}}' | grep -v amd64 | xargs -L 1 -I '{}' /bin/bash -c '%s tag "{}" "$(echo "{}" | sed s/:/-amd64:/)"'`, container.Engine, container.Engine),
```

Comment:
ditto here, this command runs inside the "node" container so it shouldn't use the host container runtime, just `"docker"` for now.

Reply:
Ok switched back to Docker, although left them as being passed in, so we could move quickly to using alternatives inside of the container.